features/datum.feature

```diff
@@ -331,85 +326,81 @@ Functionaliteit: leveren van een datum
+      Dan heeft de response een persoon met een '<danrelatie>' met de volgende '<groep>' gegevens
+      | naam                | waarde        |
+      | burgerservicenummer | true          |
+      | <veld>.type         | <type>        |
+      | <veld>.datum        | <datum>       |
+      | <veld>.jaar         | <jaar>        |
+      | <veld>.maand        | <maand>       |
+      | <veld>.onbekend     | <onbekend>    |
+      | <veld>.langFormaat  | <langFormaat> |
 
       Voorbeelden:
-      | relatie | field    | groep       | veld                 | type           | GBA InOnderzoek waarde | GbaDatum | datum      | jaar | maand | onbekend | langFormaat  |
-      | ouder   | ouders   | inOnderzoek | datumIngangOnderzoek | Datum          | 020000                 | 20200308 | 2020-03-08 |      |       |          | 8 maart 2020 |
-      | ouder   | ouders   | inOnderzoek | datumIngangOnderzoek | DatumOnbekend  | 020000                 | 00000000 |            |      |       | true     | onbekend     |
-      | ouder   | ouders   | inOnderzoek | datumIngangOnderzoek | JaarDatum      | 020000                 | 20200000 |            | 2020 |       |          | 2020         |
-      | ouder   | ouders   | inOnderzoek | datumIngangOnderzoek | JaarMaandDatum | 020000                 | 20200300 |            | 2020 | 3     |          | maart 2020   |
-      | partner | partners | inOnderzoek | datumIngangOnderzoek | Datum          | 050000                 | 20200308 | 2020-03-08 |      |       |          | 8 maart 2020 |
-      | partner | partners | inOnderzoek | datumIngangOnderzoek | DatumOnbekend  | 050000                 | 00000000 |            |      |       | true     | onbekend     |
-      | partner | partners | inOnderzoek | datumIngangOnderzoek | JaarDatum      | 050000                 | 20200000 |            | 2020 |       |          | 2020         |
-      | partner | partners | inOnderzoek | datumIngangOnderzoek | JaarMaandDatum | 050000                 | 20200300 |            | 2020 | 3     |          | maart 2020   |
-      | kind    | kinderen | inOnderzoek | datumIngangOnderzoek | Datum          | 090000                 | 20200308 | 2020-03-08 |      |       |          | 8 maart 2020 |
-      | kind    | kinderen | inOnderzoek | datumIngangOnderzoek | DatumOnbekend  | 090000                 | 00000000 |            |      |       | true     | onbekend     |
-      | kind    | kinderen | inOnderzoek | datumIngangOnderzoek | JaarDatum      | 090000                 | 20200000 |            | 2020 |       |          | 2020         |
-      | kind    | kinderen | inOnderzoek | datumIngangOnderzoek | JaarMaandDatum | 090000                 | 20200300 |            | 2020 | 3     |          | maart 2020   |
+      | relatie   | danrelatie | field       | groep                | veld                 | type           | GBA InOnderzoek waarde | GbaDatum | datum      | jaar | maand | onbekend | langFormaat  |
```

Comment:
de waarde van 'GBA InOnderzoek waarde' moet worden aangepast naar de burgerservice in onderzoek waarde. Dus 020000 --> 020120, 05000 --> 050120 en 09000 --> 090120. Dan slagen deze scenario's ook

Reply:
Ik heb de in onderzoek waardes aangepast 

Die fields-mapping-gba.csv wordt door @fsamwel gegenereerd. daar blijf ik af, maar verzoek aan Frank of hij bovenstaande opmerking van Melvin hierover kan meenemen. ALs dat gebeurd is dan neem ik aan dat de 2e opmerking van @MelvLee voldoende afgehandeld is. Zo niet, gaarne reactie.

Reply:
Issue aangemaakt om bovenstaande opmerking te borgen